types/node/node-tests.ts

```diff
@@ -87,6 +87,8 @@ namespace assert_tests {
         assert.strictEqual(1, 1, "uses === comparator");
 
         assert.throws(() => { throw new Error("a hammer at your face"); }, undefined, "DODGED IT");
+
+        assert.strict.strict.deepEqual([[[1, 2, 3]], 4, 5], [[[1, 2, '3']], 4, 5]);
```

Comment:
This test should fail:

* The purpose of assert.strict is that (at least semantically) `assert.strict.equal === assert.strictEqual` (etc.)
* `assert.strict.strict === assert.strict`

You can fix it by replacing deepEqual -> notDeepEqual

Reply:
The test won't actually get run, it's solely for type checking